qiskit/assembler/assemble_schedules.py

```diff
@@ -241,18 +241,30 @@ def _validate_meas_map(instruction_map: Dict[Tuple[int, instructions.Acquire],
         QiskitError: If the instructions do not satisfy the measurement map.
     """
     meas_map_sets = [set(m) for m in meas_map]
-
-    # Check each acquisition time individually
-    for _, instrs in instruction_map.items():
-        measured_qubits = set()
-        for inst in instrs:
-            measured_qubits.add(inst.channel.index)
-
-        for meas_set in meas_map_sets:
-            intersection = measured_qubits.intersection(meas_set)
-            if intersection and intersection != meas_set:
-                raise QiskitError('Qubits to be acquired: {} do not satisfy required qubits '
-                                  'in measurement map: {}'.format(measured_qubits, meas_set))
+    # import ipdb; ipdb.set_trace();
```

Comment:
Different cases to consider:

1. There are no overlapping acquisitions (No pair of keys `(time, duration)` with overlapping intervals) => no error
2. There are overlapping acquisitions and:
      - the two instructions acquire qubits which aren't in the same measurement group (qubits from first acquire: `[0, 1]`, qubits from second acquire: `[2, 3]`, measurement map: `[[0, 1], [2, 3]]`) => no error
      - at least one qubit is common to both acquire instructions (qubits from first acquire: `[0, 1, 2]`, qubits from second acquire: `[1]`) => error
      - the two instructions acquire different qubits but they share at least one measurement group (qubits from first acquire: `[0, 1, 2]`, qubits from second acquire: `[3, 4]`, measurement map: `[[0, 1], [2, 3], [4]]`) => error